graviti/manager/draft.py

```diff
@@ -5,17 +5,17 @@
 
 """The implementation of the Draft and DraftManager."""
 
-from typing import TYPE_CHECKING, Generator, Iterator, KeysView, MutableMapping, Optional
+from typing import TYPE_CHECKING, Generator, Optional
 
-from graviti.dataframe import DataFrame
 from graviti.manager.lazy import PagingList
+from graviti.manager.sheets import Sheets
 from graviti.openapi import create_draft, get_draft, list_drafts
 
 if TYPE_CHECKING:
     from graviti.manager.dataset import Dataset
 
 
-class Draft(MutableMapping[str, DataFrame]):
+class Draft(Sheets):
```

Comment:
Why `Draft` does not inherit `AttrsMixin, ReprMixin`?

Reply:
we will add repr in another pr